scripts/vm-vagrant.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+DPDK_DRIVER=${1:-uio_pci_generic}
+DPDK_DEVICES=${2:-0000:00:08.0}
+VAGRANT_HOME_DIR=${3:-/home/vagrant}
+
+# `modprobe` DPDK driver.
+modprobe $DPDK_DRIVER
+
+# Set vm hugepages for sysctl.
+grep -qxF "vm.nr_hugepages = 2048" /etc/sysctl.conf || \
```

Comment:
this is what I was asking earlier, wondering is it "better" to use `if; then fi` or `||` for these lines.

Reply:
@drunkirishcoder ill ask on slack.